tests/integration/WooCommerce/TransactionalEmailsTest.php

```diff
@@ -55,6 +55,30 @@ function testInitUsesImageFromWCSettings() {
     expect($email->body)->contains('my-awesome-image-url');
   }
 
+  function testGetEmailHeadings() {
```

Comment:
I have two questions: 
- Why is this an integration test and not a unit test? 
- Why did you include both changes into a single commit, those are not related changes, are they?

Reply:
- This is an integration test because it needs to call some WP functions internally (`wp_specialchars_decode` and `wp_parse_url`).
- You are right, my bad. I should have separated the typo fix into its own commit.

Reply:
> This is an integration test because it needs to call some WP functions internally (wp_specialchars_decode and wp_parse_url).
 
Didn't we create the `WPFunctions` class for this exact reason? To be able to mock WP calls ?